ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-release-4.17.yaml

```diff
@@ -7,52 +15,26 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-9-golang-1.22-openshift-4.17
-  os:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-base-openshift-4.9
-  os-minimal:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-base-openshift-4.9
 build_root:
   from_repository: true
 images:
 - dockerfile_path: ci/dockerfiles/builder.Dockerfile
   from: root
   to: osdk-builder
-- dockerfile_path: release/ansible/Dockerfile.ocp
-  from: os
-  inputs:
-    ocp_builder_rhel-8-golang-1.22-openshift-4.17:
-      as:
-      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.17
-  to: ansible-operator
 - dockerfile_path: release/helm/Dockerfile
-  from: os-minimal
+  from: base
```

Comment:
Any specific reason why helm operator uses rhel 8 and operator sdk uses rhel 9?

Reply:
The operator-sdk Dockerfile uses rhel9 image: https://github.com/openshift/ocp-release-operator-sdk/blob/d147fd3b63cf4a6bc4c1d9168f5388f2da102505/release/sdk/Dockerfile
The helm operator Dockerfile uses rhel8 image: https://github.com/openshift/ocp-release-operator-sdk/blob/d147fd3b63cf4a6bc4c1d9168f5388f2da102505/release/helm/Dockerfile

Used the same base images to keep parity with the repo Dockerfiles.

Reply:
This needs revalidation when compared to https://github.com/openshift/ocp-release-operator-sdk/commit/2fcf5f0457f4aaccf3043cd99488b1d5050deada

Reply:
The operator-sdk Dockerfile https://github.com/openshift/ocp-release-operator-sdk/commit/2fcf5f0457f4aaccf3043cd99488b1d5050deada uses rhel9 builder image and rhel8 base image. This might lead to same issues we were facing with GLIBC while building ansible-operator image. I have used rhel9 builder and base images. The `from:` will replace the last `FROM` in the Dockerfile: https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-container-images.

The helm operator images are same as that in Dockerfile of https://github.com/openshift/ocp-release-operator-sdk/commit/2fcf5f0457f4aaccf3043cd99488b1d5050deada